### DISCUSSION TITLE
Property Sets / Extended data

### DISCUSSION THREAD

**Discussion by bjarteskogoy** [DomCR/ACadSharp#433] [Q&A]
Hi,
Is it possble to read extended data property sets using ACadSharp?

**Comment by DomCR** [comment]
Hi @bjarteskogoy,

`ACadSharp` supports extended data, all implementations of `CadObject` have a property called `ExtendedData` after reading the document check for the objects with the extended data that you are  interested in.

Let me know if this is useful for you.
